md/applicationCreation.md

```diff
@@ -0,0 +1,65 @@
+# Application Descriptor Creation
+
+::: info
+**Note:** We provide a custom editor for Performance and Efficiency editions only. A xml editor is available for all editions
+:::
+
+## Overview
+
+In Bonita 7.5, we introduced the possibility to manage Application Descriptors from the Studio.  
+It is a step in our objective to have an unique deployment artifact.  
+Thus, all resources used by [Applications](applications.md) are centralized, shared by the Studio and the UIdesigner.  
+Those resources are used to create Application Descriptors from the Studio. Application Descriptors can be deployed in the Portal to create Applications.  
+
+## What's an Application Descriptor ? 
+
+Basically, an **Application Descriptor** is a XML file. It represents the skeleton of an Application, it contains all the references used by the Portal to build an Application.  
+An **Application File** is a XML file containing several Application Descriptors.  
+
+	 <!--  Application File -->
+    <applications>
+		<application>
+			<!--  First Application Descriptor -->
+		</application>
+		<application>
+			<!--  Second Application Descriptor -->
+		</application>
+	</applications>
+	
+Application Descriptors in a same Application File are totally **independent**.  
+It is interesting to have several Application Descriptors in a same file if the associated Applications are linked, but it is definitively not an obligation.  
+
+## How to create and build an Application Descriptor
+
+From the Studio, click on **_Development_** -> **_Application Descriptor_** -> **_New.._**  
+It will create an empty Application File with a default name and open an editor. You can rename this Application file from the editor or from the open menu.  
+
+![Empty Application File](images/applicationDescriptors/emptyApplicationFile.png)
+Here is an overview of the editor provided (_only for efficiency and performance editions_).  
+You can switch from the editor to the xml source at any moment, using the tabs _Editor_ and _Source_ at the bottom of the editor.  
+A toolbar is embedded to the editor, use it to:  
+
+ - **Add** an application descriptor to this Application File
+ - **Deploy** this application File (i.e all the Application Descriptors contained in this Application File) and the referenced resources (pages, layout..) in the portal. /!\ If your pages use Rest API Extensions, you have to deploy those API manually.   
+ - **Rename** this Application File
+ - **Export** this Application File
+ - **Delete** this Application File from your current workspace
+
+To create the first Application Descriptor of an Application File, use the button in the middle of the editor or the add button in the toolbar (same effect).  
+
+![Application Descriptor Editor](images/applicationDescriptors/applicationDescriptorEditor.png)
+Here is the Application Descriptor Editor provided. It is bonded to the XML source, you can use it to build your Application Descriptor without to code!  
```

Comment:
Here is the application descriptor graphical UI (editor) provided. It is bound to the XML source, so you can use it to build your application descriptor without even coding!